src/unix/linux-syscalls.c

```diff
@@ -334,21 +335,13 @@ int uv__epoll_pwait(int epfd,
                     struct uv__epoll_event* events,
                     int nevents,
                     int timeout,
-                    uint64_t sigmask) {
+                    const sigset_t* sigmask) {
 #if defined(__NR_epoll_pwait)
-  int result;
-  result = syscall(__NR_epoll_pwait,
-                   epfd,
-                   events,
-                   nevents,
-                   timeout,
-                   &sigmask,
-                   sizeof(sigmask));
-#if MSAN_ACTIVE
-  if (result > 0)
-    __msan_unpoison(events, sizeof(events[0]) * result);
-#endif
-  return result;
+  return epoll_pwait(epfd,
```

Comment:
I think @bnoordhuis suggested you drop the uv__epoll_pwait wrapper altogether and call epoll_pwait directly in linux-core.c